changelog/unreleased/pull-3203

```diff
@@ -0,0 +1,9 @@
+Enhancement: Allow to check specific pack files
+
+Restic check used to check either all packfiles or a subset that is chosen
+either in a deterministic or random way but did not allow to specify the pack
+files to check directly.
+This feature is now added. It allows to fine-tune the checking process or support
+troubleshooting in case of corrupted repositories.
```

Comment:
Suggestion:
Restic check used to check either 1) all packfiles or 2) a subset of the packfiles.
The subset was chosen either a) deterministically or b) randomly.
This feature adds the possibility to provide an explicit list of pack files, thereby making it possible to fine-tune the checking process when, for example, troubleshooting in case of corrupted repositories.